ui/src/app/utils/access-utils.spec.tsx

```diff
@@ -928,3 +929,54 @@ describe(isCompleted.name, () => {
     expect(isCompleted(status, duccSignedVersion)).toBeFalsy();
   });
 });
+
+describe(syncModulesExternal.name, () => {
+  beforeEach(() => {
+    registerApiClient(ProfileApi, new ProfileApiStub());
+    serverConfigStore.set({
+      config: {
+        ...defaultServerConfig,
+      },
+    });
+  });
+
+  it('should return a successful promise when no modules are passed', () => {
+    expect(syncModulesExternal([])).resolves.toBeTruthy();
+  });
+
+  it('should return a successful promise when all externalSyncActions succeed ', () => {
+    const spyERA = jest.spyOn(profileApi(), 'syncEraCommonsStatus');
+    spyERA.mockImplementation(() => Promise.resolve(null));
+    expect(syncModulesExternal([AccessModule.ERA_COMMONS])).toBeTruthy();
```

Comment:
oops this one too :)

Reply:
Thank you! I opened a separate PR:
https://github.com/all-of-us/workbench/pull/8163